components/Careers/ApplyNewJob.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+<div class="font-nunito text-base text-black dark:text-white dark:bg-slate-900">
+    <!-- Start Hero -->
+    <section class="relative table w-full py-36 lg:py-44 bg-[url('../../assets/images/job/job.jpg')] bg-no-repeat bg-center bg-cover">
+        <div class="absolute inset-0 bg-black opacity-80"></div>
+        <div class="container">
+            <div class="grid grid-cols-1 pb-8 text-center mt-12">
+                <h3 class="mb-4 md:text-4xl text-3xl md:leading-normal leading-normal font-medium text-white">Senior Web Developer</h3>
+
+                <ul class="list-none">
+                    <li class="inline text-slate-400 ltr:mr-3 rtl:ml-3"><i class="uil uil-map-marker text-white h6 ltr:mr-1 rtl:ml-1"></i> London, UK - <span class="text-white">Full Time</span></li>
+                </ul>
+            </div>
+            <!--end grid-->
+        </div>
+        <!--end container-->
+    </section>
+    <!--end section-->
+    <div class="relative">
+        <div class="shape absolute right-0 sm:-bottom-px -bottom-[2px] left-0 overflow-hidden text-white dark:text-slate-900">
```

Comment:
Can you please check If theme can change from header then you can remove from here

Reply:
yes its working fine from header. i have removed it from here